modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -240,7 +240,8 @@
     </thead>
     <tbody>
         {section name=item loop=$items}
-            <tr>
+		<!-- clicking row takes you to the candidate page --> 
+		<tr onclick="location.href = $(this).find('td a#pscid').attr('href');" style="cursor:pointer;">
```

Comment:
This should go in the candidate_list.js module file and candidate_list.css file based on a classname, not directly in HTML.